context/app/static/js/components/entity-search/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useSearchkit, useSearchkitQueryValue } from '@searchkit/client';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import SearchRoundedIcon from '@material-ui/icons/SearchRounded';
+import TextField from '@material-ui/core/TextField';
+
+import { useStore } from 'js/components/entity-search/SearchWrapper/store';
+
+function SearchBar() {
+  const { entityType } = useStore();
+
+  const [query, setQuery] = useSearchkitQueryValue();
+  const api = useSearchkit();
+  return (
+    <form
+      onSubmit={(event) => {
+        setQuery(query);
+        api.setQuery(query);
+        api.search();
+        event.preventDefault();
+      }}
+      role="search"
+    >
+      <TextField
+        id={`${entityType}-search`}
```

Comment:
I don't think this is referenced by the code in this PR; Referenced from somewhere else?

Reply:
`TextField` is a composition of multiple elements which all rely on the id.